app/src/main/java/com/duckduckgo/mobile/android/duckduckgo/domain/bookmark/Bookmark.java

```diff
@@ -0,0 +1,15 @@
+package com.duckduckgo.mobile.android.duckduckgo.domain.bookmark;
+
+/**
+ * Created by fgei on 6/12/17.
+ */
+
+public interface Bookmark {
+    String getId();
+
+    String getName();
+
+    String getUrl();
+
+    int getIndex();
```

Comment:
I thought it was the best way to deal with ordering the list, I see on iOS you can drag and switch items and the order is preserved when relaunching the app, since in the SharedPreference we are storing just "key - value", I though that was the easiest way to preserve that. If you have a cleaner idea, I'm open to change it and remove the index